feature/experimental/tunnel/otg_tests/tunnel_interface_based_ipv4_gre_encapsulation_test/README.md

```diff
@@ -0,0 +1,82 @@
+# TUN-1.3: Interface based IPv4 GRE Encapsulation
+
+## Summary
+
+Validate Interface based Ipv4 GRE Tunnel Config.
+
+## Procedure
```

Comment:
This is what the Readme says, why is yours different?


Validate the GRE configuration
GRE Tunnel interfaces configuration options
Tunnel source 
Tunnel source should be able to configure with unnumbered interface address
Tunnel Destination
Directly on Tunnel interface or Tunnel Group
Configure such 32 tunnel interfaces
Filter match and action
Filter is able to match the source and destination IP ranges or prefix lists
Filter is able to divert traffic to IPv4 Next HOP
Filter is able to count the packet 
Filter diverting traffic to IPv4 NH
Configure static route with IPv4 NH pointing to Tunnel interfaces
Filter application to specific interface
Apply filter on IPv4 AF on DUT-PORT1

Validate the applied config did not report any errors

Send 1000 IPv4 flows from tester on ATE-PORT1 connected to DUT-PORT1  where GRE Encap filter is applied with the traffic profile [documented](https://docs.google.com/document/d/1ByGgA92QjUZm7HMuFU9CqrcdxbT-7WA1Qh-lLDCtZ1g/edit?usp=sharing):
IPv4 flows are made of 5 tuples:
Source IP address
Source port
Destination IP address
Destination port
Payload protocol

Enable the packet capture on ATE ingress port to verify the GRE header of uncapped traffic

Verify the filter counters for packet being diverted or sent for encapsulation

Verify the tunnel interfaces counters to confirm the traffic encapsulation

After encapsulation, traffic should be load balanced/hash to all available L3 ECMP or LAG or combination of both features

Validate system for:
Health-1.1
No feature related error or drop counters incrementing, discussion with vendors required to highlight additional fields to monitor based on implementation and architecture

Reply:
Updated README without filter based content and paths as this is an interface based tunnel